drivers/hdc1000/hdc1000.c

```diff
@@ -126,6 +149,19 @@ int hdc1000_get_results(const hdc1000_t *dev, int16_t *temp, int16_t *hum)
 
 int hdc1000_read(const hdc1000_t *dev, int16_t *temp, int16_t *hum)
 {
+    /* Return cached data to mitigate redundant sensing */
```

Comment:
The API says `This function will trigger a new conversion, wait for the conversion to be finished and the get the results from the device.` IMO we should **definitely not** use cached data here.